frontend/lib/types.ts

```diff
@@ -69,7 +69,7 @@ export enum StatusSuggestionStatus {
  * This is the full object returned by a get to the Student endpoint
  */
 export type StudentData = {
-  collection: StudentBase[];
+  collection: StudentBaseList[];
   totalLength: number;
 };
```

Comment:
I need to move this to a separate StudentDataList type instead of changing the base type now that we have multiple views.